cli_player/cli_player.py

```diff
@@ -15,12 +16,25 @@ def make_move(
             action_result: str,
             available: list[str],
     ):
-        print(f'Делаю ход от лица {self.number}')
-        print(f'Мои хп: {my_hp}')
-        print(f'Хп оппонента: {opponent_hp}')
-        print(f'Мои вещи: {my_items}')
-        print(f'Вещи оппонента: {opponent_items}')
-        print(f'Действие: {action}')
-        print(f'Результат действия: {action_result}')
-        print(f'Доступные ходы: {available}')
-        return input('Мой ход: ')
+        print(self.language.acting_player(), end='')
+        print(self.number)
+        print(self.language.hp(), end='')
+        print(my_hp)
+        print(self.language.opponent_hp(), end='')
+        print(opponent_hp)
+        print(self.language.my_items(), end='')
+        print(my_items)
+        print(self.language.opponent_items(), end='')
+        print(opponent_items)
+        print(self.language.action(), end='')
+        print(action)
+        print(self.language.action_result(), end='')
+        print(action_result)
+        if len(available) > 0:
+            print(self.language.available_moves(), end='')
+            print(available)
+            move = input(self.language.my_move())
+        else:
+            move = 'NOTHING'
```

Comment:
Note: This represents a change in behavior not related to language. It auto-picks "NOTHING" if there is not a valid option.